lib/codit/population/networks/city.py

```diff
@@ -88,13 +100,17 @@ def build_households(people):
     n_individuals = len(people)
     assigned = 0
     households = []
-
     num_h = int(n_individuals / AVERAGE_HOUSEHOLD_SIZE)
     household_examples = build_characteristic_households(num_h)
+    # create num_h of homes
+    homes_list = build_households_home_list(num_h)
+    logging.info(f"There are {len(homes_list)} households generated for accommodation buildings")
 
     while assigned < n_individuals:
         ages = next_household_ages(household_examples)
         size = len(ages)
+        # randomly pick up a home from list of homes
+        home = next_household_home(homes_list)
```

Comment:
maybe this variable, `home`, could be called `home_specification`?

Reply:
Yes, home_specification is better name for coordinates and building_types